src/themes/material/assets/md/cell.edit.scss

```diff
@@ -220,8 +220,8 @@ td.q-grid-edit {
 
 		img {
 			position: relative;
-			max-height: 100%;
-			max-width: 100%;
+			max-height: 100% !important;
```

Comment:
What was the issue? Maybe instead of important 'display: inline-block' could help?

Reply:
the issue is the change of editor now styles can assume that it located on html root but this is not true at the moment, i believe this is temporary solution